docs/pages/installation.mdx

```diff
@@ -12,6 +12,17 @@ guide](./get-started.mdx).
 
 </Admonition>
 
+<Admonition type="warning" title="Teleport client compatibility">
+
+When installing Teleport clients - `tsh`, `tctl`, and `tbot` - ensure to
+install the same major version as the Teleport cluster. For example, if the cluster is
+running Teleport 12, you should install a Teleport 12 client.
```

Comment:
The docs here should match what's listed on our component compatibility page, which is that clients can be 1 major version behind.

Maybe we should say something like this:
```suggestion
For best results, Teleport clients (`tsh`, `tctl`, `tbot`)
should be the same major version as the cluster they are
connecting to. Teleport servers are compatible with clients
that are on the same major version or one major version older.
Teleport servers do not support clients that are on a newer
major version.
```